### DISCUSSION TITLE
How to update the Adobe Air on windows to run the .air file?

### DISCUSSION THREAD

**Discussion by tidus2020** [airsdk/Adobe-Runtime-Support#606] [Q&A]
I packaged .air file with the latest version AIR33 SDK, but it can't run success on the windows and mention to update the AIR SDK.

**Comment by ajwfrost** [comment]
We've not yet got a shared runtime available for desktop, and are recommending that people use the "bundle" approach where you package the runtime together with the application.
https://help.adobe.com/en_US/air/build/WSfffb011ac560372f709e16db131e43659b9-8000.html
